### PR TITLE
Meeting minutes, 2020-05-07

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2020/2020-05-07-coordination.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+## News, general matters
+
+### Letter of support for EuSSI project
+
+Our CERN colleagues (Jose Gonzalez, Tim Smith) asked for a letter of support for their EuSSI proposal, a H2020 call. The project abstract is [here](https://drive.google.com/open?id=18Tua2QiQ9JKi7Vw9Bgdn3lP9qm7ffb5U) and they aim at building a Europe-wide initiative for Research Software with the goal of software becoming a first-class citizen in research. The initiative will provide guidelines for good software sustainability practices and reproducible science and integrate with many of the similar existing local initiatives (NLeSC, SSI, etc.).
+
+This looks very well aligned with the HSF and we have prepared a [draft letter of support](https://docs.google.com/document/d/1Pqc918XiqmrcGuPCZKjgi8QPxqUEZNgVKaaBXZRnq6c/edit?usp=sharing).
+
+Q. What about software standards? What constitures "well written software"? A. Yes, there is a work package on sustainability and quality. There will be a body that tries to improve this area and also training is a significant work package.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2020/2020-05-07-coordination.md:18]:
```suggestion
Q. What about software standards? What constitutes "well written software"? A. Yes, there is a work package on sustainability and quality. There will be a body that tries to improve this area and also training is a significant work package.
```

File: organization/_posts/2020/2020-05-07-coordination.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+* On the tools side we're going to be adding an optional GPU monitoring support to `prmon` (see issue [#105](https://github.com/HSF/prmon/issues/105)). The main idea is to initially support NVidia and use `nvidia-smi` for collection. Then extend it to support other hardware as well.
+
+
+### Software Training
+
+- Will organise virtual training soon. Recorded talks, with small group work a few days after.
+
+### Event generators
+* Updating the generator paper. Will submit to the journal next week (still waiting for some comments).
+    * Also plan to contribute it to the Snowmass process.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2020/2020-05-07-coordination.md:133]:
```suggestion
    * Also plan to contribute to the Snowmass process.
```

File: organization/_posts/2020/2020-05-07-coordination.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+### PyHEP 2020 Workshop
+
+[Virtual Workshop](https://indico.cern.ch/e/pyhep2020).
+
+## AOB
+
+### Next Meeting
+
+- Next regular meeeting slot is 21 May, *however this is the Ascension holiday in many European countries (and at CERN)* so no meeting.
```
Comment (amete) [inline organization/_posts/2020/2020-05-07-coordination.md:160]: meeeting => meeting

File: organization/_posts/2020/2020-05-07-coordination.md (hunk 2 above)
Comment (graeme-a-stewart) [reply organization/_posts/2020/2020-05-07-coordination.md:133]: "it" = "the generator paper", which I think is ok

**Comment by graeme-a-stewart** [comment]
URL check failures not to do with this PR

**Pull request merged**